cloudaux/orchestration/aws/iam/managed_policy.py

```diff
@@ -9,40 +9,40 @@
 from cloudaux.decorators import modify_output
 from flagpole import FlagRegistry, Flags
 
+from cloudaux.orchestration.aws import _conn_from_args
+from cloudaux.orchestration.aws.iam import MissingFieldException
 
 registry = FlagRegistry()
 FLAGS = Flags('BASE')
 
 
 @registry.register(flag=FLAGS.BASE)
-def get_base(policy_arn, **conn):
+def get_base(managed_policy, **conn):
```

Comment:
This change seems a little out of scope. Are we getting something substantive out of it?

Reply:
It's to be consistent with IAM Roles and Users.